lib/money/currency.rb

```diff
@@ -24,14 +25,28 @@ def find(currency_iso)
       def currencies
         @@currencies ||= Loader.load_currencies
       end
+
+      def normalize_currency(currency_code)
+        code = currency_code.to_s.downcase
+        return code if currencies.keys.include?(code)
+
+        code = currency_normalization_map[code.upcase]
```

Comment:
I want to revisit the entire currency database soon by having it pull data from currency-iso.org and generate JSON from there. It's incomplete and incorrect (e.g. HUF as a fractionless currency). I want to fix the casing there to be upcase because that's what is normally used.